src/ServicePulse.Host/app/js/services/services.monitoring.js

```diff
@@ -2,27 +2,36 @@
 (function (window, angular, $, undefined) {
     'use strict';
 
-    function Service($http, scConfig, notifications, uri) {
-
-        function getRaw() {
-            var url = uri.join(scConfig.monitoring_url, '/raw');
-            return $http.get(url)
-                .then(function (response) {
-                    return response.data;
-                })
-                .catch(function(fallback) {
-                    notifications.pushSticky('Can not connect to Monitoring Service');
+    function Service($http, rx, scConfig, uri) {
+
+        function getEndpoints() {
+            return rx.Observable.merge(scConfig.monitoring_urls.map(function (url) {
+                var requestUri = uri.join(url, '/raw');
+
+                var httpRequest = rx.Observable.just(url)
+                    .flatMap(function (requestUrl) {
+                        return $http.get(requestUrl);
+                    });
+
+                var repeatRequests = rx.Observable.interval(5000).flatMapLatest(httpRequest.retry().delay(5000));
```

Comment:
Am I understanding this correctly that we will retry the call indefinitely? Do we want to log it somehow when the call has failed?